### PR TITLE
Fixes #1

### LINKED ISSUE #1: reduce redundancy in URL conversion.
Current behavior is that even if URL already exists in database, still a new entry is added. Reduce redundant URL's and return the shortened URL already in database.

### PR DESCRIPTION
**Pull request by RishiRaj22**

### CODE REVIEW

**Comment by RishiRaj22** [comment]
@zamhaq Have you tested it yet?

**Comment by zamhaq** [comment]
Sorry @RishiRaj22 my Semester exams are going on. I will check by tonight.

**Comment by zamhaq** [comment]
Also it would be great if you could follow the pull request template and provide info about what you changed in your pr.

File: server/server.js [hunk 1]
```diff
@@ -20,18 +20,23 @@ app.get('/new/*',(req,res) => {
...
       const protocol = req.protocol;
-      Math.seedrandom(original_url);
-      const now = Date.now();
-      const rnum = Math.floor(Math.random() * (now/10000000));
-      const short_url = protocol + '://' + hostname + '/' + rnum;
-      var newurl = new url({
-            original_url,
-            rnum,
-            short_url
+      url.findOne({original_url: original_url},(err,foundUrl) => {
```
Comment (zamhaq) [inline server/server.js]: you can use the short notation if `key` and `value` have same name i.e `{original_url}`

**Comment by RishiRaj22** [comment]
Ok, it was my first time so I didn't know about PR template. I forgot to add return after sending the new url. I have taken all your feedback into consideration. Should I send a new PR with these changes or is there any other way around? I have no prior experience with this stuff.

**Comment by RishiRaj22** [comment]
@zamhaq I hope this will do it. Just needed to return from the function after sending the existing URL.

**Pull request merged**